### ISSUE TITLE
Nodes and Nps are not updated when the engine doesn't report them alongside with PV

### ISSUE THREAD

**Issue by alex65536** [jhonnold/node-tlcv#23]
When playing, my UCI engine doesn't send the number of nodes alongside with PV, but reports it separately. For example, the output may look as follows:

```
go infinite
...
info depth 11 time 1160 pv d2d4 g8f6 b1c3 d7d5 g1f3 b8c6 e2e3 c8f5 f1d3 f5d3 d1d3 score cp 57
info depth 12 time 2996 pv e2e4 e7e5 g1f3 g8f6 b1c3 b8c6 f1d3 f8d6 e1g1 e8g8 c3b5 d6c5 score cp 16
info nodes 18567391 time 3010 nps 6166569
info depth 13 time 4993 pv e2e4 e7e5 g1f3 d7d5 f3e5 b8d7 e5d7 c8d7 e4d5 f8d6 f1c4 d8h4 d1e2 score cp 42
info nodes 36638104 time 6022 nps 6083159
info nodes 54337370 time 9004 nps 6034702
```

Still, web UI shows `Nodes: 0.00` instead of the reported number.

I am not sure whether it's an issue in `node-tlcv` or the underlying TLCV server.

**Comment by jhonnold** [comment]
**TLDR** This is an issue with the server.

---

The [nodes and time](https://github.com/jhonnold/node-tlcv/blob/main/src/handler.ts#L117-L118) come on the PV command from the server. My imagination is that the pv command is sent whenever a pv is printed by the engine, but not for other cases, thus resulting in the behavior you see. 

I cannot confirm this as I do not own the server code.

**Comment by alex65536** [comment]
I can confirm now that it's really the issue with TLCV server, as other clients also do not show Nodes and Nps for me. Sorry for disturbing.
